### ISSUE TITLE
Drag attachment from Outlook and upload

### ISSUE THREAD

**Issue by nheath99** [microsoft/AzureStorageExplorer#2963]
**Is your feature request related to a problem? Please describe.**
I often get files emailed to me from colleagues, which are to be uploaded to blob storage using Storage Explorer. If I try and drag the attachment from the Outlook message to Storage Explorer, nothing happens.

**Describe the solution you'd like**
When I drag a file from Outlook to Storage Explorer it is uploaded the same way as if I was to drag a file from File Explorer.

**Describe alternatives you've considered**
The current workaround is to save the attachment locally and then upload the file from the saved location. Another alternative is to grant my colleagues access to Azure Storage, but they are not technical staff and I would be concerned about them breaking something.

**Additional context**
I think that the way Outlook saves attachments in a temp directory is different to other programs, which is why drag-drop doesn't work to 3rd party programs, though I'm no expert and don't fully understand the issue.

**Comment by jinglouMSFT** [comment]
Good idea, but I'd be a bit more careful in terms of drag-and-drop from outlook attachment directly to Azure Storage. If we add the functionality, we will need to consider the cases where the attachment contains malware or harmful stuff.

**Comment by nheath99** [comment]
Yes that seems like a reasonable consideration. Perhaps there could be a setting where the user can specify what file types to allow drag-and-drop? In my scenario the files are images.
